src/main/java/com/vocabsvc/wordtrainer/model/Users.java

```diff
@@ -1,19 +1,27 @@
 package com.vocabsvc.wordtrainer.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.*;
 import lombok.Getter;
 import lombok.Setter;
 
-import javax.persistence.*;
-
 @Getter
 @Setter
 @Entity
 @Table
 public class Users {
```

Comment:
Prefer entity names to be singular, User over Users.

Reply:
thanks for pointing it out missed that, user was already an inbuilt table in postgres so used Users as a quick fix.